### PR TITLE
Use 'id' key to fetch the task id from within a kafka task payload

### PR DESCRIPTION
**Pull request by eclarizio**
~~This is necessary because the task update kafka messages can potentially have either task_id or id as the identification key in the payload.~~

"task_id" is the incorrect key that should be coming back from the kafka events, and we should be using "id" instead.

@syncrou @gmcculloug 

Related PRs:
>Topo Persister PR that precipited this change: ?>https://github.com/RedHatInsights/topological_inventory-persister/pull/56/files
>
>Topo Operations PR that allows this change to work for either operations / persister based events:
>RedHatInsights/topological_inventory-api#301

### CODE REVIEW

**Review by gmcculloug — APPROVED** [review]
LGTM

@syncrou Can you link to the PR that introduced this change (moving completion task to topo persister) and the new PR to change "task_id" to "id" on the operations worker.

We need to hold off merging until we have the Topology PR ready to merge.

**Comment by syncrou** [comment]
Topo Persister PR that precipited this change: https://github.com/RedHatInsights/topological_inventory-persister/pull/56/files

Topo Operations PR that allows this change to work for either operations / persister based events:
https://github.com/RedHatInsights/topological_inventory-api/pull/301

**Review by syncrou — APPROVED** [review]
LGTM 👍

**Pull request merged**
